docs/octopus-rest-api/tentacle.exe-command-line/index.md

```diff
@@ -22,7 +22,7 @@ Tentacle supports the following commands:
 - **[import-certificate](/docs/octopus-rest-api/tentacle.exe-command-line/import-certificate.md)**:  Replace the certificate that Tentacle uses to authenticate itself.
 - **[list-instances](/docs/octopus-rest-api/tentacle.exe-command-line/list-instances.md)**:  Lists all installed Tentacle instances.
 - **[new-certificate](/docs/octopus-rest-api/tentacle.exe-command-line/new-certificate.md)**:  Creates and installs a new certificate for this Tentacle.
-- **[polling-proxy](/docs/octopus-rest-api/tentacle.exe-command-line/polling-proxy.md)**:  Configure the HTTP proxy used by polling Tentacles to reach the Octopus Server.
+- **[polling-proxy](/docs/octopus-rest-api/tentacle.exe-command-line/polling-proxy.md)**:  Configure the HTTP proxy used by polling tentacles to reach the Octopus Server.
```

Comment:
Ahh - found that it wasn't fixed at source - fixed now - https://github.com/OctopusDeploy/OctopusTentacle/commit/01031985a38085b7c1bfbf3cda546f652cd06cbb

Reply:
Looks like we'll be shipping another Tentacle [soon](https://octopusdeploy.slack.com/archives/C072C2D5L/p1595546171016000?thread_ts=1595306286.012500&cid=C072C2D5L)